xspec/assembly/enduser-files.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
what is the role of this folder? 
I see many technical folders and files at the top level of the repo. Can we hide them somehow?